src/mad_spark_alt/evolution/strategy_comparison.py

```
@@ -0,0 +1,458 @@
+
+            # Get fitness histories from generation snapshots
+            all_histories = []
+            for run in result.runs:
+                if run.generation_snapshots:
+                    history = [snap.best_fitness for snap in run.generation_snapshots]
+                    all_histories.append(history)
+                else:
+                    all_histories.append(
+                        [run.final_best_fitness]
+                        if hasattr(run, "final_best_fitness")
+                        else [0.0]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)